src/main/resources/assets/computercraft/lua/rom/programs/delete.lua

```diff
@@ -8,7 +8,7 @@ end
 local function isMountPoint(path)
     if not fs.isDir(path) then
         return false
-    elseif fs.getDrive(path) == fs.getDrive(fs.combine(path, "..")) then
+    elseif fs.combine(path, "..") == ".." or fs.getDrive(path) == fs.getDrive(fs.combine(path, "..")) then -- the == ".." check prevents errors when path is "/"
```

Comment:
I would like to remove the need for this comment but I'm not sure how clear this code is on it's own